### ISSUE TITLE
oops + shadow-cljs 2.12 are not working together because of old GCL 

### ISSUE THREAD

**Issue by n2o** [binaryage/cljs-oops#26]
Heyho, thanks for this project, it makes javascript interop a lot better than using native functions!

I realized, that you are already replacing old GCL functions, and [shadow-cljs released a new version today](https://github.com/thheller/shadow-cljs/commit/733f61bc040ab5ef93ddbc2b63d58b233be34292), which breaks using oops + latest shadow-cljs together. And I hope that you can release a new version soon :-)

Thanks for your work in this project!

**Comment by darwin** [comment]
Hi, I believe current master has fixes for recent GCL update. Could you test it with your project? I would then release a new cljs-oops version for convenience.

**Comment by n2o** [comment]
Hi, thanks for the quick reply. Yes, your latest commit in your master branch a2b48d59047c28decb0d6334e2debbf21848e29c fixes the problems.

Tested with latest versions of shadow-cljs and clojurescript:
```
org.clojure/clojurescript {:mvn/version "1.10.844"}
thheller/shadow-cljs {:mvn/version "2.12.1"}
```

**Comment by darwin** [comment]
Thanks for confirmation. Fixed in [v0.7.1](https://github.com/binaryage/cljs-oops/releases/tag/v0.7.1).
